Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -558,5 +581,21 @@ private void FillMonitorAction()
 
             Log.Trace("GDAXBrokerage.FillMonitorAction(): task ended");
         }
+
+        /// <summary>
+        /// Checks if this brokerage supports the specified symbol
+        /// </summary>
+        /// <param name="symbol">The symbol</param>
+        /// <returns>returns true if brokerage supports the specified symbol; otherwise false</returns>
+        public bool CanSubscribe(Symbol symbol)
```

Comment:
nit - Think this method can be private?

Reply:
moved to `GDAXDataQueueHandler`, and now it can be private.
done